backend/src/main/java/kr/momo/exception/GlobalExceptionHandler.java

```diff
@@ -24,6 +27,17 @@ public ProblemDetail handleMomoException(MomoException ex) {
         return new CustomProblemDetail(problemDetail, ex.errorCode());
     }
 
+    @ExceptionHandler
+    public ProblemDetail handleMethodArgumentNotValidException(MethodArgumentNotValidException ex) {
+        log.warn(ex.getMessage(), ex);
+
+        List<String> errors = ex.getBindingResult().getFieldErrors().stream()
+                .map(error -> "%s:%s".formatted(error.getField(), error.getDefaultMessage()))
+                .toList();
+        ProblemDetail problemDetail = ProblemDetail.forStatusAndDetail(HttpStatus.BAD_REQUEST, errors.toString());
+        return new CustomProblemDetail(problemDetail, INVALID_REQUEST_FORMAT_MESSAGE);
+    }
+
```

Comment:
다음과 같이 에러들을 메세지로 변환하는 로직을 메서드 분리를 해보면 가독성이 조금 더 좋아질 것 같습니다!

```suggestion
    @ExceptionHandler
    public ProblemDetail handleMethodArgumentNotValidException(MethodArgumentNotValidException ex) {
        log.warn(ex.getMessage(), ex);

        List<String> errors = getErrorsToMessage(ex);
        ProblemDetail problemDetail = ProblemDetail.forStatusAndDetail(HttpStatus.BAD_REQUEST, errors.toString());
        return new CustomProblemDetail(problemDetail, INVALID_REQUEST_FORMAT_MESSAGE);
    }

    private List<String> getErrorsToMessage(MethodArgumentNotValidException ex) {
        return ex.getBindingResult()
                .getFieldErrors()
                .stream()
                .map(error -> "%s:%s".formatted(error.getField(), error.getDefaultMessage()))
                .toList();
    }
```